app/design/frontend/Elogic/maksym/Magento_Catalog/web/js/qty-counter.js

```diff
@@ -0,0 +1,30 @@
+define(["jquery"], function ($) {
+    ("use strict");
+
+    $(function () {
+        $("button.adds").on("click", add);
+        $("button.subs").on("click", remove);
+    });
+
+    function add() {
+        var input = $("input.qty"),
+            value = input.val();
+
+        input.val(++value);
+
+        if (value > 0) {
+            $("button.subs").removeAttr("disabled");
+        }
+    }
+
+    function remove() {
+        var input = $("input.qty"),
+            value = input.val();
+
+        if (value > 1) {
+            input.val(--value);
+        } else {
+            $("button.subs").attr("disabled", "disabled");
+        }
+    }
+});
```

Comment:
Your JS is still just a JS code, you need to declare a widget:

```
$.widget('maksym.qtyControls', {
        _create: function() {
                 // Your code goes here
        }
});

return $.maksym.qtyControls;
```

Reply:
Done / noted with thanks.